src/HS/UserBundle/Controller/DefaultController.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace HS\UserBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class DefaultController extends Controller
+{
```

Comment:
@Reddaa y'aurait pas une bonne pratique pour commenter les méthodes ?

Reply:
It's a great idea, take a look at https://www.phpdoc.org/  , It implements Javadoc. and easy to use, you generate the doc with comment ; what do you guys think?